moveLighting.js

kastion opened the pull request:
Submitting an API script used to move dynamic lighting path objects between layers. Useful when combined with other scripts such as TokenMod to Open/Close doors by moving the dynamic lighting path object to the gmlayer when opening a door (And using TokenMod to move the door token to gmlayer) then using this script to move the dynamic lighting path object back to the walls layer (And using TokenMod to move the door token from gmlayer back to objects or map layer)

With this simple script you can easily manage doors and toggle line of sight on/off of areas blocked with dynamic lighting path objects.

Solarswordsman commented:
This script is cool, but a bit of your formatting is off.  Don't include previous versions if you don't have them in the folder structure, Roll20 will not find them :P.  You can just put an empty array there since this is a new script (as far as Roll20's one-click is concerned).  Lastly, put your moveLighting.js in a folder with the version number as its name (`0.1.4`).

kastion commented:
I updated the json and moved the moveLighting.js file to
"moveLighting/0.1.4/moveLighting.js"

I'll take care of the json for PatrolNPC as soon as I am able to.

On Tue, May 8, 2018 at 12:16 PM, Jeffrey Lamb <notifications@github.com>
wrote:

> This script is cool, but a bit of your formatting is off. Don't include
> previous versions if you don't have them in the folder structure, Roll20
> will not find them :P. You can just put an empty array there since this is
> a new script (as far as Roll20's one-click is concerned). Lastly, put your
> moveLighting.js in a folder with the version number as its name (0.1.4).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Roll20/roll20-api-scripts/pull/620#issuecomment-387438216>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/Ak-BhwA7eBaODkTtd7IzsmIWe4uEN6k-ks5twba9gaJpZM4TuJwL>
> .
>

Solarswordsman commented:
Just as a heads up, best practice is to be using `\n` for new lines, so I went ahead and changed that on your description.  I'll keep an eye out for the PatrolNPC as soon as you get to it.  Thanks!
